core/src/runtime.rs

```diff
@@ -214,6 +217,39 @@ impl Runtime {
         WeakRuntime(Ref::downgrade(&self.inner))
     }
 
+    pub fn set_interrupt_handler(&self, handler: Option<Box<dyn FnMut() -> bool + 'static>>) {
```

Comment:
Please describe this function using doc-comment.

Reply:
Ops, I overlooked it. It's included now.